.github/workflows/update-flake-lock.yml

```diff
@@ -0,0 +1,19 @@
+name: update-flake-lock
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 7 9 * *'
```

Comment:
This is oddly specific. Something like weekly, I would understand. But 7 AM of every 9th day of month???

Reply:
This is Guilhem's code I am merging humble-devel into refactor + few commits.